### ISSUE TITLE
This hotfix appears to be ready...

### ISSUE THREAD

**Issue by tbohn** [UW-Hydro/VIC#57]
Hi Joe,

I believe this hotfix is ready for release.  I had asked you to wait on it last week, in light of Homero's finding that melt_energy was getting garbage values intermittently (well, a single occurrence in a single grid cell, out of several grid cells).  We couldn't find any concrete evidence that this problem was caused by any of the fixes in the hotfix.

Just as an aside: we ran VIC in valgrind to check for memory issues that could possibly cause garbage to pop up in unrelated variables.  VIC came out clean.

Since Homero has opened a separate issue regarding melt_energy, it would seem that melt_energy can be dealt with in a separate hotfix.  Of course it is your and Bart's call.

Ted
